lua/nordic/groups/integrations.lua

```diff
@@ -97,39 +97,83 @@ function M.get_groups()
     G.MiniStatuslineFileinfo = { bg = LC.normal.b.bg, fg = LC.normal.b.fg }
     G.MiniStatuslineInactive = { bg = LC.inactive.a.bg, fg = LC.inactive.a.fg, bold = LC.inactive.a.gui == 'bold' }
 
+    -- Tree Groups
+    G.TreeNormal = { fg = C.fg, bg = C.bg }
+    G.TreeNormalNC = { fg = C.fg, bg = C.bg }
+    G.TreeRootFolder = { fg = C.gray4 }
+    G.TreeRootName = { fg = C.white2 }
+    G.TreeFileIcon = { fg = C.blue2 }
+    G.TreeFileNameOpened = { fg = C.fg }
+    G.TreeSpecialFile = { fg = C.magenta.bright }
+    G.TreeGitConflict = { fg = C.magenta.bright }
+    G.TreeGitModified = { fg = C.git.change }
+    G.TreeGitDirty = { fg = C.gray4 }
+    G.TreeGitAdded = { fg = C.git.add }
+    G.TreeGitNew = { fg = C.gray4 }
+    G.TreeGitDeleted = { fg = C.gray4 }
+    G.TreeGitStaged = { fg = C.gray4 }
+    G.TreeGitUntracked = { fg = C.orange.base }
+    G.TreeTitleBar = { fg = C.gray2, bg = C.white3 }
+    G.TreeFloatBorder = { fg = C.white3, bg = C.gray0 }
+    G.TreeCursorLine = { bg = C.gray1 }
```

Comment:
what about using `bg_selected` here its already what is used for `G.PmenuSel = { bg = C.bg_selected } -- Popup menu: selected item.`

Reply:
all I changed was the ones I mentioned in the last comment. I didn't actually check how the other colors looked.

Reply:
I just "grouped" them as alex told me his idea

Reply:
I meant to only select the `TreeCursorLine`...

Reply:
oh okay. so as far as I can understand you want to only change the color of TreeCursorLine right?

Reply:
> I just "grouped" them as alex told me his idea

I did not see your message. If you would like we can change that it a different PR.



> oh okay. so as far as I can understand you want to only change the color of TreeCursorLine right?

Correct.

Reply:
I can change that color. You want to change it to C.bg_selected right?

Reply:
i don't actually know if I get what you mean here

> I did not see your message. If you would like we can change that it a different PR.

Reply:
@5-pebbles should I change the TreeCursorLine bg to C.bg_selected and push it?

Reply:
Alright: GitHub is not showing your messages unless I refresh the page which is weird. So I did not notice that you had commented on my message.

I think this line: `G.TreeCursorLine = { bg = C.gray1 }` should be changed to this `G.TreeCursorLine = { bg = C.bg_selected }`.

However it was like this before your pull request was opened and so I was offering to open a different PR to fix it. After yours has been merged.

Reply:
> @5-pebbles should I change the TreeCursorLine bg to C.bg_selected and push it?

yes

Reply:
done

Reply:
> However it was like this before your pull request was opened and so I was offering to open a different PR to fix it. After yours has been merged.
 That would be no problem for me but since I pushed it now, this is no more needed right?

Reply:
great thank you! ... but your formatter did the thing again... lol

Reply:
oh shit forgot to turn it off hahahahaha
i'll fix it now

Reply:
> > However it was like this before your pull request was opened and so I was offering to open a different PR to fix it. After yours has been merged.
> 
> That would be no problem for me but since I pushed it now, this is no more needed right?

Correct

Reply:
fixed it